gofiles/gofiles.go

```diff
@@ -97,3 +85,36 @@ func Write(dir string, files []GoFileSpec) (map[string]GoFile, error) {
 
 	return goFiles, nil
 }
+
+// WriteRaw writes the Go files represented by the specifications in the files parameter using the
+// provided directory as the root directory.
+//
+// Unlike Write, WriteRaw does not enforce the requirement for a valid go file structure and will simply
+// write the specified files as is to the root directory, returning any errors that may be encountered.
+func WriteRaw(dir string, files []GoFileSpec) error {
```

Comment:
Naming is hard, especially with no overloading, so I opted for this but am certainly open to other options.